qemu/tests/cfg/tcpreplay.cfg

```diff
@@ -0,0 +1,7 @@
+- tcpreplay:
+    image_snapshot = yes
+    type = tcpreplay
+    tcpreplay_file_name = "tcpreplay-4.3.1.tar.gz"
+    pcap_file_name = "2603.pcap"
+    tcpreplay_compile_cmd = "cd /tmp && rm -rf tcpreplay-4.3.1 && tar -xzvf ${tcpreplay_file_name} && cd tcpreplay-4.3.1 && ./configure && make && make install"
```

Comment:
there are libraries available in Python and Avocado providing the functionalities you needed, please use libraries instead of executing commands with `avocado.utils.process`:
* `shutil` to deal with files and directories copy, move, etc.
*  `avocado.utils.archive` to deal with compressed files.